modules/profiles/release.nix

```diff
@@ -18,6 +18,8 @@ in
       # TODO this needs to be refined when we define a policy for the
       # processes and the UID/groups that should be enabled by default
       # if not already covered by systemd
-      ghaf.users.accounts.enable = true;
+
+      users.mutableUsers = false;
+      users.users.root.hashedPassword = "!";
```

Comment:
we probably need to keep root even in release until we have another remote update mechanism in place, suggest to comment `users.users.root.hashedPassword = "!";` out for now